src/nvim/options.lua

```diff
@@ -8729,6 +8729,7 @@ local options = {
           '%<',
           '%f %h%w%m%r ',
           '%=',
+          "%{% luaeval(\"package.loaded['vim.status'] and vim.status.get_progress_status() or ''\") %}",
```

Comment:
Note(self): This should fix the test error in channel_spec. However, now since by default vim. status is lazy loaded it doesn't get activated at all by default. Need to figure something out. vim.status should be lazy-loaded. But now the question is what triggers it to activate.

Reply:
@justinmk anyidea what can be done for this? Reading the conversation on the diagnostinc-status pr and it's followups this seems to be a limiation in our current module system.